src/main/java/com/example/Java6thTopic/Average.java

```diff
@@ -0,0 +1,22 @@
+package com.example.Java6thTopic;
+
+import java.util.List;
+
+public class Average {
+
+    private final List<Parson> members;
+    private final com.example.Java6thTopic.AverageText averageText;
```

Comment:
importしておくと以下のように記述できます
```suggestion
    private AverageText averageText;
```